src/schema.rs

```diff
@@ -818,18 +818,13 @@ fn parsing_canonical_form(schema: &serde_json::Value) -> String {
         serde_json::Value::Object(map) => pcf_map(map),
         serde_json::Value::String(s) => pcf_string(s),
         serde_json::Value::Array(v) => pcf_array(v),
-        serde_json::Value::Number(n) => pcf_number(n),
```

Comment:
Would you mind explaining why you are removing this line? It isn't immediately clear to me.

Reply:
Well as this was used only for default values, if default values are stripped away then this become unnecessary.